adapters/Snap/SnapAdapter/GADMediationAdapterSnap.m

```diff
@@ -14,71 +14,115 @@
 
 #import "GADMediationAdapterSnap.h"
 
+#import "GADMediationSnapBanner.h"
+#import "GADMediationSnapRewardedAd.h"
+#import "GADMediationSnapInterstitial.h"
 #import "GADMediationAdapterSnapConstants.h"
 
-@implementation GADMediationAdapterSnap
+#import <SAKSDK/SAKSDK.h>
 
-- (void)collectSignalsForRequestParameters:(nonnull GADRTBRequestParameters *)params
-                         completionHandler:(nonnull GADRTBSignalCompletionHandler)completionHandler {
-  // TODO: Start signal generation and call completionHandler when the signal generation finishes or fails.
-  completionHandler("", nil);
+@implementation GADMediationAdapterSnap {
+    GADMediationSnapBanner *_banner;
+    GADMediationSnapInterstitial *_interstitial;
+    GADMediationSnapRewardedAd *_rewardedAd;
```

Comment:
Nit: Can rename `_rewardedAd` to `_rewarded` to maintain consistency, or rename:
- `_banner` to `_bannerAd`
- `_interstitial` to `_interstitialAd`